restapi/endpoints_test.go

```diff
@@ -0,0 +1,146 @@
+package restapi
+
+import (
+	"fmt"
+	"net/http"
+	"github.com/spf13/viper"
+	"net/http/httptest"
+	"testing"
+	"github.com/gavv/httpexpect/v2"
+	"github.com/stretchr/testify/suite"
+	"github.com/uwblueprint/shoe-project/internal/database/migrations"
+	"github.com/uwblueprint/shoe-project/internal/database/models"
+	"github.com/uwblueprint/shoe-project/testutils"
+	"gorm.io/gorm"
+)
+
+type endpointTestSuite struct {
+	suite.Suite
+	endpoint *httpexpect.Expect
+	db       *gorm.DB
+	token     string
+}
+
+func (suite *endpointTestSuite) SetupSuite() {
+	db, err := testutils.MockDatabase()
+	if err != nil {
+		suite.Fail("error while creating database", err)
+	}
+	suite.db = db
+
+	// setup required for jwt authentication
+	viper.SetDefault("auth.jwt_key", "GotFKl1PGgMpLg7D36NiI0hy/gsl6woTCXYdhKATbzc=")
+	viper.SetDefault("auth.jwt_expiry", "2h")
```

Comment:
nit: if we're logging in on every test setup, why do we need the token expiry to be this long?

Reply:
Hm yeah you're right, could also just keep it as 2h to be consistent?

Reply:
consistent with?

Reply:
Since the real token expiry is 2h?

Reply:
yeah maybe set it as `5m` that should be fine